### PR TITLE
Polymer 3 2019 01 31 103116

### PR DESCRIPTION
**Pull request by m6jones**

### CODE REVIEW

File: test/d2l-course-image.html [hunk 1]
```diff
@@ -3,7 +3,6 @@
 	<head>
 		<meta charset="utf-8">
 		<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
-		<script src="../../@babel/polyfill/browser.js"></script>
```
Comment (m6jones) [inline test/d2l-course-image.html]: Looking into this change. Might have to add it to the patch?

File: package.json [hunk 2]
```diff
@@ -25,17 +25,16 @@
...
   "devDependencies": {
-    "@babel/polyfill": "^7.0.0",
-    "@polymer/iron-component-page": "^3.0.0-pre.18",
-    "@polymer/iron-demo-helpers": "^3.0.0-pre.18",
-    "@polymer/iron-test-helpers": "^3.0.0-pre.18",
-    "@webcomponents/webcomponentsjs": "^2.2.1",
+    "@polymer/iron-component-page": "^4.0.0",
+    "@polymer/iron-demo-helpers": "^3.0.0",
+    "@polymer/iron-test-helpers": "^3.0.0",
+    "@webcomponents/webcomponentsjs": "^2.2.4",
```
Comment (m6jones) [inline package.json]: Are these changes expected?

File: d2l-course-image.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-course-image">
 			}
 		</style>
 
-		<img src="[[_src]]" srcset$="[[_srcset]]" sizes$="[[_tileSizes]]" on-load="_showImage" class$="[[_imageClass]]">
+		<img src="[[_src]]" srcset$="[[_srcset]]" sizes$="[[_tileSizes]]" on-load="_showImage" class$="[[_imageClass]]" alt="" aria-hidden="">
```
Comment (m6jones) [inline d2l-course-image.js]: The expected change.

File: test/d2l-course-image.html (hunk 1 above)
Comment (dlockhart) [reply test/d2l-course-image.html]: This change is OK -- we changed the converter to no longer add it.

File: package.json (hunk 2 above)
Comment (dlockhart) [reply package.json]: Yeah, I would expect the converter to always pull in the latest.

**Pull request merged**